### PR TITLE
Completed css-brochure-form exercise. Edwin Tan

### PR DESCRIPTION
**Pull request by edtan094**
image

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Edwin! your form is too wide as compared to the example, please revise and resubmit with a new screenshot

**Comment by edtan094** [comment]
image
Screenshot attached

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+div.div1 {
+  width: 37%;
+  margin: auto;
+}
+
+div.div2 {
+  text-align: right;
+  width: 37%;
+  margin: auto;
+}
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css]: These class names aren't very descriptive of their purpose, try to use a name that better articulates what they are for.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great work! One note for you:

**Review by codingcodymiller — APPROVED** [review]
Great work!!

**Pull request merged**